### PR TITLE
update undeploy.json with draft entry

### PR DESCRIPTION
**Pull request by LotharBender**

### CODE REVIEW

File: guides/databases-hana.md [hunk 1]
```diff
@@ -558,7 +558,8 @@ By default, `cds add hana` creates an `undeploy.json` like this:
 [
   "src/gen/**/*.hdbview",
   "src/gen/**/*.hdbindex",
-  "src/gen/**/*.hdbconstraint"
+  "src/gen/**/*.hdbconstraint",
+  "src/gen/**/*_drafts.hdbtable"
```
Comment (renejeglinsky) [inline guides/databases-hana.md:562]: Is this already valid or for the upcoming release?

Comment (LotharBender) [reply guides/databases-hana.md:562]: Yes, has already been merged - see https://github.tools.sap/cap/cds-tools/issues/1106

Comment (renejeglinsky) [reply guides/databases-hana.md:562]: Ok, it's merged. But it's not yet released, right? There hasn't been a release in the past 5 days.

Comment (LotharBender) [reply guides/databases-hana.md:562]: Maybe I misunderstood your initial question: 
> Is this already valid or for the upcoming release?

It is valid for the upcoming release 7.6.0 - I have to admit that I am little bit confused now ;-)
Do I have to set some specific label?

Comment (renejeglinsky) [reply guides/databases-hana.md:562]: In general the "do not merge" label would be good. But as we are now approaching our next release, there's no need to hold it back. We can merge already.

**Review by renejeglinsky — APPROVED** [review]

**Pull request merged**